### PR TITLE
Fix TF2 Estimator batch size for predict

### PR DESCRIPTION
**Pull request by hkvision**
For predict, each worker should use the local batch instead of the global batch, to align with fit and evaluate.

### CODE REVIEW

File: python/orca/src/bigdl/orca/learn/tf2/pyspark_estimator.py [hunk 1]
```diff
@@ -330,7 +332,9 @@ def predict(self, data, batch_size=None, verbose=1,
         :param data: predict input data.  It can be XShards or Spark DataFrame.
                If data is XShards, each partition can be a Pandas DataFrame or a dictionary of
                {'x': feature}, where feature is a numpy array or a tuple of numpy arrays.
-        :param batch_size: Batch size used for inference. Default: None.
```
[the remark this answers is not in the document]
Comment (jason-dai) [reply python/orca/src/bigdl/orca/learn/tf2/pyspark_estimator.py:333]: > 

How do we handle None for prediction?

[the remark this answers is not in the document]
Comment (hkvision) [reply python/orca/src/bigdl/orca/learn/tf2/pyspark_estimator.py:333]: Just input None to keras_model.predict, I suppose keras will handle it and use 32?

[the remark this answers is not in the document]
Comment (jason-dai) [reply python/orca/src/bigdl/orca/learn/tf2/pyspark_estimator.py:333]: Then we can just set it to 32.

**Review by lalalapotter — COMMENTED** [review]
LGTM

**Pull request merged**